2022-12-26 09:29:59.562 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration frigate which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-12-26 09:29:59.563 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-12-26 09:30:04.015 ERROR (MainThread) [custom_components.hacs] HACS is disabled - invalid_token
2022-12-26 09:30:04.017 ERROR (MainThread) [custom_components.hacs] Bad credentials
Traceback (most recent call last):
  File "/config/custom_components/hacs/base.py", line 457, in async_can_update
    response = await self.async_github_api_method(self.githubapi.rate_limit)
  File "/config/custom_components/hacs/base.py", line 511, in async_github_api_method
    raise HacsException(_exception)
custom_components.hacs.exceptions.HacsException: Bad credentials
2022-12-26 09:31:17.148 ERROR (MainThread) [frontend.js.latest.202212130] :0:0 ResizeObserver loop completed with undelivered notifications.
2022-12-26 09:33:55.270 ERROR (MainThread) [frontend.js.latest.202212130] :0:0 ResizeObserver loop completed with undelivered notifications.
2022-12-26 09:34:10.592 ERROR (MainThread) [frontend.js.latest.202212130] :0:0 ResizeObserver loop completed with undelivered notifications.
2022-12-26 09:34:10.642 ERROR (MainThread) [frontend.js.latest.202212130] :0:0 ResizeObserver loop completed with undelivered notifications.
2022-12-26 09:34:10.664 ERROR (MainThread) [frontend.js.latest.202212130] :0:0 ResizeObserver loop completed with undelivered notifications.
2022-12-26 09:34:11.192 ERROR (MainThread) [frontend.js.latest.202212130] :0:0 ResizeObserver loop completed with undelivered notifications.
2022-12-26 09:34:42.052 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry  for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 372, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 431, in async_setup_entry
    return await component.async_setup_entry(entry)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 166, in async_setup_entry
    raise ValueError("Config entry has already been setup!")
ValueError: Config entry has already been setup!
2022-12-26 09:34:42.882 ERROR (MainThread) [custom_components.hacs] <Integration blakeblackshear/frigate-hass-integration> HttpStatusCode.UNAUTHORIZED
2022-12-26 09:34:42.895 ERROR (MainThread) [custom_components.hacs] <Integration hacs/integration> HttpStatusCode.UNAUTHORIZED
2022-12-26 09:34:42.908 ERROR (MainThread) [custom_components.hacs] <Integration blakeblackshear/frigate-hass-integration> HttpStatusCode.UNAUTHORIZED
2022-12-26 09:34:42.910 ERROR (MainThread) [custom_components.hacs] <Integration hacs/integration> HttpStatusCode.UNAUTHORIZED
2022-12-26 09:34:42.946 ERROR (MainThread) [custom_components.hacs] HACS is disabled - invalid_token
2022-12-26 09:34:44.497 ERROR (MainThread) [custom_components.hacs] <QueueManager> No manifest.json file found 'custom_components/None/manifest.json'
2022-12-26 09:34:44.499 ERROR (MainThread) [custom_components.hacs] <QueueManager> No manifest.json file found 'custom_components/None/manifest.json'
2022-12-26 09:34:46.028 ERROR (MainThread) [custom_components.hacs] <QueueManager> No manifest.json file found 'custom_components/None/manifest.json'
2022-12-26 09:34:46.031 ERROR (MainThread) [custom_components.hacs] <QueueManager> No manifest.json file found 'custom_components/None/manifest.json'
